src/skin/popup.css

```diff
@@ -249,6 +249,12 @@ font-size: 16px;
   padding-right: 5px;
   padding-top: 7px;
 }
+#share{
+  float: right;
+  padding-right: 5px;
+  padding-top: 7px;
+  cursor: pointer;
```

Comment:
Do we need to set the "cursor"?

Reply:
Yes; because it's not a "proper" link (doesn't have an `href`), it doesn't get styled by `a:-webkit-any-link`.